project_tag/views/project.xml

```diff
@@ -58,4 +58,19 @@
         </xpath>
     </field>
   </record>
+
+
+<!-- Search view -->

```

Comment:
```suggestion
 <!-- Search view -->
```

Fix indentation

Reply:
My mistake: there were here two missing blank characters, I added only one so now is still missing the other one  :see_no_evil: :smile: